pkg/cronjob/spark_dependencies.go

```diff
@@ -50,6 +52,12 @@ func CreateSparkDependencies(jaeger *v1.Jaeger) *batchv1beta1.CronJob {
 
 	commonSpec := util.Merge([]v1.JaegerCommonSpec{jaeger.Spec.Storage.Dependencies.JaegerCommonSpec, jaeger.Spec.JaegerCommonSpec, baseCommonSpec})
 
+	image := jaeger.Spec.Storage.Dependencies.Image
+	if image == "" {
```

Comment:
Why isn't this file using `util.ImageName`? The  `es_index_cleaner` is using is in this PR.

Reply:
The `util.ImageName` resolves the image name but also appends the current Jaeger version. As the dependencies image is the only one that does not have a tag, the image is just resolved here.

Reply:
:+1: could you please add a comment to the source code?

Reply:
done